src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -131,7 +130,7 @@ func (cf *fetcher) fetchCloudConfig(cfg *Config) ([]byte, error) {
 	// successive requests
 	req.Close = true
 
-	resp, err := cf.httpFetcher.Do(req)
+	resp, err := cf.rt.RoundTrip(req)
```

Comment:
This is actually kind of amazing that just calling RoundTrip essentially in place of Do accomplishes the same thing. We don't technically need the HTTP client itself at all here - nice!